challenge_7/java/zmiller91/MissingNumber.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * @author zmiller
+ */
+public class MissingNumber {
+    
+    public static int find(int[] input) {
+        
+        // Invalid
+        if (input.length == 0) {
+            return -1;
+        }
+        
+        // Find the sum, min, and max values
+        int max = Integer.MIN_VALUE;
+        int actualSum = 0;
+        for(int i : input) {
+            actualSum += i;
+            max = Math.max(max, i);
```

Comment:
Solution is fixed and README.md is added.  Those READMEs seem redundant and unnecessary.